imgix/constants.py

```diff
@@ -8,6 +8,22 @@
 SRCSET_INCREMENT_PERCENTAGE = 8
 SRCSET_MAX_SIZE = 8192
 
+# The percentage by which image width-values are incremented by for
+# use in generating the default set of image widths, i.e.
+# `[IMAGE_MIN_WIDTH, .., IMAGE_MAX_WIDTH]`.
+INCREMENT_PERCENTAGE = 0.08
```

Comment:
@jayeb and I have discussed this at length in the past, but we specifically want to avoid labeling this as an increment percentage moving forward. According to our documentation, "the `srcset` width tolerance dictates the maximum tolerated size difference between an image's downloaded size and its rendered size". I recommend we label this (and change any instances similar to `increment_percentage`) to `SRCSET_WIDTH_TOLERANCE`.

Reply:
Devil's advocate:

This is a percentage though. We use the value to increment each subsequent image width by the given percentage. Perhaps a better name might be `IMAGE_WIDTH_TOLERANCE`. I have found the language you quoted above in our rails and imgix-core-js docs. _Where did this language come from?_ Is it standardize somewhere? Is it used in the [Source Set](https://html.spec.whatwg.org/multipage/images.html#source-set) spec?

Names are only as good as their ability to describe what they refer to, we should aim to use good names where we can and be consistent where we can't.

Reply:
They can both be percentages, in this case we're talking about the maximum tolerable difference _as a percentage_. If you look in the source code itself, `100` to `116` is not an 8 percent jump.
The definition comes from us internally, the spec does not concern itself (from what I can tell) with how width descriptors are spaced out.
> Names are only as good as their ability to describe what they refer to

I agree, for that reason I think it is important to specifically call out `SRCSET`.

Reply:
What does "tolerate" mean in this context? And,
>100 to 116 is not an 8 percent jump.

this is right, so could you give me an example of

>maximum tolerable difference as a percentage

as it relates to the language/name we should use?

Reply:
What do you think about `SRCSET_WIDTH_TOLERANCE`?

Reply:
I do like `SRCSET_WIDTH_TOLERANCE` if you like it. I like it more than `SRCSET_INCREMENT_PERCENTAGE`.

Reply:
@sherwinski the latest commit confirms this